lang/driver/src/database.rs

```diff
@@ -354,6 +355,16 @@ impl Database {
         ir
     }
 
+    // Core API: js
```

Comment:
This does not correspond to the Core API, which is concerned with handling the content of caches. See the comment above in this file.

Reply:
Would you prefer not adding it to the driver for now?